.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: Build and test
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build-macos:
+    runs-on: macos-latest
+    steps:
+      # Checks-out the repo. More at: https://github.com/actions/checkout
+      - uses: actions/checkout@v2
+      - name: Select Xcode version
+        run: sudo xcode-select -switch /Applications/Xcode_11.4.1.app
+      - name: Test in Debug
+        run: swift test -c debug
+
+  build-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: Didstopia/SwiftAction@v1.0.2
+        with:
+          swift-action: test --enable-test-discovery
```

Comment:
I don't have much experience with it, but ubuntu-latest looks to have pre-installed Swift.
https://github.com/actions/virtual-environments/blame/master/images/linux/Ubuntu1804-README.md#L40
Do we still need to use Docker-ed one, to avoid any problem with it? Maybe to fix the version?

Reply:
Yeah, the benefit of the Docker image is that it specifies the exact OS version and Swift verions, while with the `ubuntu-latest` image we're at the mercy of GitHub Actions maintainers, who can update those versions without (or with little) warning. While here, we know that `Didstopia/SwiftAction@v1.0.2` ships with Swift 5.2, `Didstopia/SwiftAction@v1.0.1` with Swift 5.1, we can even test the library with both versions, which I do in some of my own repositories to make sure that backward and forward compatibility is maintained.